src/cgr_gwas_qc/workflow/scripts/grouped_contamination.py

```diff
@@ -98,12 +138,13 @@ def estimate_contamination(
 
 if __name__ == "__main__" and "snakemake" in locals():
     main(
-        **{k: v for k, v in snakemake.input.items() if not k.startswith("_")},  # type: ignore # noqa
-        grp=snakemake.params.grp,  # type: ignore # noqa
-        gtc_pattern=snakemake.params.gtc_pattern,  # type: ignore # noqa
-        snps=int(snakemake.params.snps),  # type: ignore # noqa
-        conda_env=snakemake.params.conda_env,  # type: ignore # noqa
-        notemp=bool(snakemake.params.notemp),  # type: ignore # noqa
-        outfile=snakemake.output[0],  # type: ignore # noqa
-        threads=snakemake.threads or 2,  # type: ignore # noqa
+        defaults = {}
+        defaults.update({k: Path(v) for k, v in snakemake.input.items() if not k.startswith("_")})  # type: ignore # noqa
```

Comment:
Looks like Black is failing on this line. Can you address this?
<img width="1315" alt="image" src="https://github.com/user-attachments/assets/f39827c4-4166-47a3-9a1d-8a0ecdaa7806">

Reply:
2b8acbcd6056ded1b2e2a135051761bc094a0dd6 should fix this. Thanks.